gadget/gadget.go

```diff
@@ -238,6 +238,14 @@ type DiskVolumeDeviceTraits struct {
 	// the volume that may be useful in identifying whether a disk matches a
 	// volume or not.
 	Structure []DiskStructureDeviceTraits `json:"structure"`
+
+	// Size is the physical size of the disk, regardless of usable space
+	// considerations.
```

Comment:
we should remember that some partition might be grown later though?

Reply:
The disk itself won't be grown? This is just for the physical volume/partition not for the partition

Reply:
@pedronis I will land this without addressing this since it will unblock things for me, but please let me know when you're back if I'm missing something here